api/v1alpha1/drclusterpeers_types.go

```diff
@@ -34,44 +34,45 @@ const (
 	Async ReplicationPolicyType = "async"
 )
 
-// ClusterPeersSpec defines the desired state of ClusterPeers
-type ClusterPeersSpec struct {
+// DRClusterPeersSpec defines the desired state of DRClusterPeers
+type DRClusterPeersSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 
-	// Members of the ClusterPeers set
-	ClusterIDs []string `json:"clusterIDs"`
+	// Members of the DRClusterPeers set
+	ClusterNames []string `json:"clusterNames"`
 
-	// Type of replication policy used by this ClusterPeers: sync or async
+	// Type of replication policy used by this DRClusterPeers: sync or async
 	ReplicationPolicy ReplicationPolicyType `json:"replicationPolicy"`
```

Comment:
(question)  Should we drop this in a subsequent cleanup, although this is a future proofing field, we can add it when required?

Reply:
Right.  Not needed now.

Reply:
Done